app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -418,11 +418,12 @@ TestOwnership getTestOwnership(String builderName, BuilderType type, String test
 
 /// Gets the [BuilderType] of the builder by looking up the information in the
 /// ci.yaml.
-BuilderType getTypeForBuilder(String? builderName, YamlMap ci) {
-  final List<dynamic>? tags = _getTags(builderName, ci);
+BuilderType getTypeForBuilder(String? builderName, CiYaml ciYaml) {
+  final List<dynamic>? tags = _getTags(builderName, ciYaml);
```

Comment:
You can update `model/ci_yaml/target.dart` to have a `get tags => ...` field

Reply:
Oooh good point.